sdk/appcenter-data/src/main/java/com/microsoft/appcenter/data/Data.java

```diff
@@ -375,7 +375,22 @@ public void run() {
     }
 
     private synchronized void processPendingOperations() {
```

Comment:
```suggestion
    @WorkerThread
    private synchronized void processPendingOperations() {
```

Would need an import though.